changeColor.js

```diff
@@ -0,0 +1,13 @@
+const divs = document.querySelectorAll("div"),
+      buttons = [];
+let square = "";
+for (const div of divs) {
+  if (div.id === "square") {
+    square = div;
+  } else {
+    div.addEventListener(
+      "mouseover",
+      () => (square.style.backgroundColor = div.id)
```

Comment:
Tak jak zapisałeś funkcja zwraca wartość assignmentu; czyli `undefined`.

Raczej chcemy zapisać to tak jak niżej żeby wprost pokazać, że to co zwraca ta funkcja nas nie interesuje.
```suggestion
      () => {
          square.style.backgroundColor = div.id;
      }
```

Reply:
słusznie, nie miałem świadomości, że te zapisy się różnią w działaniu - dosłownie z godzinę temu, przy refactoringu mrcoffee, zwróciłem na to uwagę